.github/scripts/instr-count-benchmark/run-benchmark.sh

```diff
@@ -13,3 +13,6 @@ PYTORCH_GIT_VERSION=$(python -c "import torch; print(torch.version.git_version)"
 echo "Running instruction benchmark for pytorch-${PYTORCH_VERSION}, commit SHA: ${PYTORCH_GIT_VERSION}"
 
 # Run the instruction count benchmark
+pushd ${REPO_CHECKOUT}
+python main.py --mode ci --destination ${RESULT_JSON}
```

Comment:
Assuming `REPO_CHECKOUT` is the pytorch repository base, I think we should run `benchmarks/instruction_counts/main.py`?

Reply:
They expect to be run from `benchmarks/instruction_counts`. (It affects import paths. e.g. `from core.api` rather than `from benchmarks.instruction_counts.core.api`)

Reply:
But then shouldn't we first enter the `${BENCHMARK_ROOT}` directory?
```suggestion
python main.py --mode ci --destination ${BENCHMARK_ROOT}
```